src/components/ZetkinCalendar/index.tsx

```diff
@@ -51,10 +67,29 @@ const ZetkinCalendar = ({ baseHref, events, campaigns , tasks }: ZetkinCalendarP
         }
     };
 
+
+    const today = new Date();
+    const { firstDayInView, lastDayInView } = getViewRange(focusDate, range);
+
+    const isTodayBeforeView = () =>{
+        return today < firstDayInView;
+    };
+
+    const isTodayAfterView = () =>{
+        return today > lastDayInView;
+    };
+
+    const handleTodayBtnClick = () => {
+        setFocusDate(new Date());
+    };
+
     return (
         <Box display="flex" flexDirection="column" height={ 1 }>
             <Box display="grid" flexGrow={ 0 } gridTemplateAreas={ `". nav view"` } gridTemplateColumns="repeat(3, minmax(0, 1fr))">
                 <Box alignItems="center" display="flex" gridArea="nav" justifyContent="center">
+                    {
+                        isTodayBeforeView() ? <Tooltip arrow placement="top" title="Today"><Button className={ classes.today } onClick={ handleTodayBtnClick }></Button></Tooltip> : <Button className={ classes.hide } disabled></Button>
```

Comment:
This is a really long and hard to read line. It could probably be simplified greatly, but should at least be broken up on several lines.

Reply:
I tried having Tooltip outside the ternary in different possible ways. It kept the tooltip visible all the time (like the way it was showing it when we were pair coding together.) Eventually, I only broke it up into multiple lines.